spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -243,11 +243,8 @@ def in_person_failed_fraud
     )
   end
 
-  def in_person_please_call
-    UserMailer.with(user: user, email_address: email_address_record).in_person_please_call(
-      enrollment: in_person_enrollment_id_ipp,
-      visited_location_name: in_person_visited_location_name,
-    )
```

Comment:
Wait, were `enrollment` and `visited_location_name` just not used? I was going to question their removal here, but they're actually not in the email template.

Reply:
👃 👈